db/migrate/20220114232339_add_name_to_user.rb

```diff
@@ -0,0 +1,11 @@
+class AddNameToUser < ActiveRecord::Migration[7.0]
```

Comment:
Migration name is `AddNameToUser ` but your adding not only `name` but also `role`, `belt` and `birth_date` 🤔